server/sonar-web/src/main/js/components/issue/views/set-type-form-view.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import $ from 'jquery';
+import _ from 'underscore';
+import ActionOptionsView from '../../common/action-options-view';
+import Template from '../templates/issue-set-type-form.hbs';
+
+export default ActionOptionsView.extend({
+  template: Template,
+
+  getType () {
+    return this.model.get('type');
+  },
+
+  selectInitialOption () {
+    return this.makeActive(this.getOptions().filter(`[data-value="${this.getType()}"]`));
+  },
+
+  selectOption (e) {
+    const issueType = $(e.currentTarget).data('value');
+    this.submit(issueType);
+    return ActionOptionsView.prototype.selectOption.apply(this, arguments);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use the rest syntax to declare this function's arguments. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3513)